### DISCUSSION TITLE
Ibax fake account

### DISCUSSION THREAD

**Discussion by likablex** [IBAX-io/go-ibax#1853] [Ideas]
### Describe the bug

Şuan çok sayıda fake hesap ile airdroba katılan insanlar var. Bu konu twitter'da konusulup tavsiye ediliyor. Sisteme ip kontrolu getirilmesi gerekiyor

### Reproduction

Fake account control

### System Info

```shell
Fake
```

### Validations

- [X] Read the [docs](https://docs.ibax.io).
- [X] Check that there isn't [already an issue](https://github.com/IBAX-io/go-ibax/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/IBAX-io/go-ibax/discussions) or join our [Discord Chat Server](https://discord.com/invite/zRX6Mwafya).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by caddyfor4** [comment]
I agree with u
